adafruit_trellism4.py

```diff
@@ -82,8 +83,53 @@ def __setitem__(self, index, value):
             raise IndexError("Pixel assignment outside available coordinates.")
 
         self._neopixel[offset] = value
+        if self._auto_write:
+            self.show()
```

Comment:
Not in this case because the Trellis M4 module turns off auto_write in the Neopixel module when it's initialized. I was trying to keep that the case and therefore implemented the logic at this level.